project/src/components/app/room/room.jsx

```diff
@@ -23,14 +27,20 @@ function renderCard(data) {
 }
 
 function Room(props) {
-  const {images, isPremium, title, rating, isFavorite, type, bedrooms, maxAdults, price, description, goods, host: {avatarUrl, isPro, name}} = props.hotels[0];
-  const {comment, rating: score} = props.comment;
-  const {avatarUrl: avatarAuthor, name: nameAuthor} = props.reviews;
-  const {hotels, location: {pathname}} = props;
-  const CardItem = hotels.slice(0, 3).map(renderCard);
+  const {offers, location: {pathname}} = props;
+
+  const idViewedRoom = pathname.split(':')[1];
```

Comment:
деструкутризация массивов.
```suggestion
  const [, idViewedRoom] = pathname.split(':');
```

ну и еще, ты рассмотри внимательно объект Location, явно, ты сможешь найти там параметр айди